common/persistence/PersistenceManager.php

```diff
@@ -131,4 +132,35 @@ private function createPersistence($persistenceId)
         $driver = $this->propagate(new $driverClassName());
         return $driver->connect($persistenceId, $config);
     }
+
+    /**
+     * Return a collection of all SQL schemas
+     */
+    public function getSqlSchemas() : SchemaCollection
+    {
+        $schemas = new SchemaCollection();
+        foreach (array_keys($this->getOption(self::OPTION_PERSISTENCES)) as $id) {
+            $persistence = $this->getPersistenceById($id);
+            if ($persistence instanceof \common_persistence_SqlPersistence) {
```

Comment:
Can we move this to use block?
```suggestion
            if ($persistence instanceof common_persistence_SqlPersistence) {
```